### DISCUSSION TITLE
런칭데이 피드백 모음

### DISCUSSION THREAD

**Discussion by Jaymyong66** [woowacourse-teams/2024-code-zap#572] [QA]
# 런칭데이 피드백

**Comment by zeus6768** [comment]
### 레디
- 태그를 눌렀을 때 OR 조건으로 검색되었으면 좋겠어요

### 러쉬
- 진짜 쓸 것 같다
- 익스텐션 사용할 때 제목 입력하고 싶어요
- 단축키 안내가 있으면 좋겠다. 

### 우주
- psvm 안 되나요? 
  - intellij 단축어

### 명오
- 다른 사람 코드를 검색해서 보고 싶어요.

### 프람
- 코드를 직접 작성, 개행 시 indent가 불편하다. 
- 동적인 인덴트 기능이 추가되면 좋겠어요. 
  - 카테고리 생성 시 엔터를 눌러야 하는 게 헷갈린다. 

### 잉크
- 구경가기에서 인기많은 순(조회수, 좋아요?)으로 정렬해서 보고 싶어요. 

### 아르
- "전체보기" 화면에서 검색 input이 변경될 때마다 fetch가 가서인지 반응이 조금 느린 것 같아요!
- "템플릿" 화면에서 이전 화면 이동 버튼이 있으면 좋겠어요!
  - + 제스쳐로 뒤로 돌아가기 기능이 가능하면 좋겠다. 

### 해쉬

### 쿠키
- 태그에 컴마 추가 시 분리되어도 좋을 것 같아요.
- 템플릿 복제도 좋을 것 같아요. 

### 파슬리
- 태그를 눌렀을 때 계속 이동해서 조~~금 보기가 힘들었어요. 

### 초롱
- 상세페이지에서 태그 누르면 바로 검색 되게 해주세요. 

### 예니
- 코틀린 지원해주세요!
  - 언어 하이라이트

### 로빈
- ID 특수문자 안 된다고 알려주세요.


### 구구
- 이전 로그인 사용자의 정보(태그목록) 남아있는 경우 있음. 로그아웃 시 전부 날릴 것 ㅠㅠ
- 목록조회에서 특정 템플릿 조회 후 목록조회로 돌아가면 페이지 리셋
  - 돌아가면 항상 1페이지인 게 불편하다. 

### 미분류
- 태그를 모두 해제하는 버튼이 있으면 좋겠어요. 
- 코드 작성창의 '+' 버튼이 더 크면 좋겠어요.
- 여러 모듈로 나뉘는 템플릿을 서술식으로 관리하고 싶어요. 
- 로그아웃했을 때 여전히 수정 버튼이 눌림, 수정이 되지는 않음.
- 카테고리를 삭제했을 때 새로고침이 안됨.
- 아이디 에러메시지 수정
- 비밀번호 모든 특수문자 가능함
- 카테고리 개수 제한
- 카테고리 편집할 때 연필 버튼이 아니라 그냥 누르면 수정되게 해주세요.
- 태그에 이모지 안되게 했으면 좋겠다.
- 템플릿 제목 길이 제한
- 전체 템플릿 검색 조회 기능
- 카테고리 길이 제한
- 태그 몇 개가 숨겨져 있는지 나왔으면 좋겠다 (ex. +3)
- 모든 태그 용도를 잘 모르겠다
- 삭제할 게 없으면 전체삭제 버튼 안 보여도 될 것 같다.
- 익스텐션 알림 누르면 바로 이동되면 좋을 것 같아요.
- 템플릿 생성 눌렀을 때 바로 커서가 제목쪽으로 가서 입력됏으면 좋겠다.
- 소스코드 순서를 드래그 앤 드롭으로 이동할 수 있으면 좋을 것 같다.
- 소스 저작권에 대해 고려해보면 좋을 것 같아요.
-
